### PR TITLE
Port to Orion

### PR DESCRIPTION
**Pull request by DusanJovic-NOAA**
Associated PRs:
https://github.com/ufs-community/ufs-weather-model/pull/105
https://github.com/NOAA-EMC/NEMS/pull/53

### CODE REVIEW

**Review by junwang-noaa — APPROVED** [review]

File: gfsphysics/GFS_layer/GFS_driver.F90 [hunk 1]
```diff
@@ -705,7 +705,7 @@ subroutine GFS_stochastic_driver (Model, Statein, Stateout, Sfcprop, Coupling, &
 ! Since IPD_func0d_proc declares all arguments as intent(inout), we
 ! need to do the same here - however, this way we are loosing the
 ! valuable information on the actual intent to this routine. *DH
-#ifdef __GFORTRAN__
+!#ifdef __GFORTRAN__
```
Comment (climbfuji) [inline gfsphysics/GFS_layer/GFS_driver.F90]: You might as well just remove those blocks entirely ...

**Review by climbfuji — APPROVED** [review]

File: gfsphysics/GFS_layer/GFS_driver.F90 (hunk 1 above)
Comment (junwang-noaa) [reply gfsphysics/GFS_layer/GFS_driver.F90]: I can add the code changes in my next commit. Let's commit it as is.

Comment (DusanJovic-NOAA) [reply gfsphysics/GFS_layer/GFS_driver.F90]: @climbfuji Do you want me to leave your comments about arguments mismatch or delete them?

Comment (climbfuji) [reply gfsphysics/GFS_layer/GFS_driver.F90]: I think it is best to delete the comments when we remove the code that you commented out entirely.

**Pull request merged**